.env

```diff
@@ -7,7 +7,7 @@ REACT_APP_API_HOSTNAME="https://localhost:8087"
 # REACT_APP_ALLOW_DEV_TOOLS=false
 
 # Google Analytics' Universal ID
-ANALYTICS_ID="UA-208192118-1"
+ANALYTICS_ID=
```

Comment:
I decided to remove this from the repo because the testing property will be difficult to maintain updated with the production/beta one. I think it's better to just use your own testing property if you need